NEWS.md

```diff
@@ -89,6 +89,8 @@
 
 * Increase the default `nbin` of `guide_colourbar()` to place the ticks more precisely (#3508, @yutannihilation).
 
+* `geom_sf()` no more ignores alpha on linestring geometry (#3589, @yutannihilation).
```

Comment:
Maybe write in a positive way: "`geom_sf()` now applies alpha to linestring geometries ..."?

Reply:
Thanks, it sounds better.